app/src/main/java/com/ecosystem/kin/app/MainActivity.java

```diff
@@ -105,14 +174,38 @@ private void createNativeSpendOffer() {
         }
     }
 
-    private Callback<String> getNativeSpendCallback() {
+    /**
+     * Use this method with the offerID you created, to get {@link OrderConfirmation}
+     * @param offerID
+     */
+    private void getOrderConfirmation(@NonNull final String offerID) {
```

Comment:
not used

Reply:
it's for example, we are not adding a UI for that.